src/parse/parsers.js

```diff
@@ -8,7 +8,7 @@ import frontMatter from 'front-matter';
 
 var parsers = {
   content: {
-    pattern: '\.(html|hbs|handlebars)$',
```

Comment:
@calebeby  I'm curious on the thought behind changing the RegEx pattern here (and the few below)? Was this a linting change?

Reply:
Yup. ESLint was complaining because the dot did not need to be escaped.

https://eslint.org/docs/rules/no-useless-escape

Reply:
Oh actually, if this is being passed into the Regexp constructor somewhere, the backslash itself probably has to be escaped (`"\\."`)

Reply:
Fixed ✔️

Reply:
@calebeby I'm not convinced this needs a double backslash. If anything, the patterns could be simplified to:
```js
pattern: '(html|hbs|handlebars)$'
```
> Yup. ESLint was complaining because the dot did not need to be escaped.

I'm not sure this applies as a RegExp pattern. The `\.` means that it matches the "." (period) character literally. (e.g., `.html`)

If the goal of this PR is to only fix linting issues, I don't think this change should be a part of that. 

I'm open for further discussion, though, if I'm not understanding correctly. 😉

Reply:
The backslash was ignored as part of the string, if you go to a js console and type "\.asdf" you will get back ".asdf" (the backslash does nothing because the dot does not need to be escaped)

So when this gets passed into the regex constructor, there is no backslash. The regex parser interprets the . as an any matcher, not as a literal dot.

I would rather not change the pattern to be '(html|hbs|handlebars)$', because that means that if you had a `asdfasdfhbs` file, it would match, even though that is not the extension.

If we want the matcher to match files that end with `.(html|hbs|handlebars)`, we have to ensure that a literal backslash is passed into the regex constructor. In order to put a literal backslash in a string, you have to escape it.

More info here:
- https://stackoverflow.com/questions/17863066/why-do-regex-constructors-need-to-be-double-escaped
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#Description

The reason I included this in this PR is because eslint warned with this rule: https://eslint.org/docs/rules/no-useless-escape, which was triggered because there was no reason to escape the dot.

Reply:
<img width="177" alt="screen shot 2018-07-06 at 2 00 35 pm" src="https://user-images.githubusercontent.com/13206945/42400065-08fe25ca-8125-11e8-865a-5aa00f438333.png">

Reply:
Excellent, thanks for that explanation @calebeby! 🎓 Double backslash it is! 😉